tools/charm_module_patches/src/Parallel/Algorithms/AlgorithmArray.def.h.patch

```diff
@@ -0,0 +1,49 @@
+Distributed under the MIT License.
```

Comment:
I think we do the super consistent thing (there appears to be no sarcasm emoji? :disappointed: ) of `CamelCasing` nested directories (`CharmModulePatches`)